Source/OxyPlot/Axes/LogarithmicAxis.cs

```diff
@@ -207,8 +207,7 @@ public override void Pan(ScreenPoint ppt, ScreenPoint cpt)
         /// <returns>The value.</returns>
         public override double InverseTransform(double sx)
         {
-            // Inline the <see cref="PostInverseTransform" /> method here.
-            return Math.Exp((sx / this.Scale) + this.Offset);
+            return this.PostInverseTransform((sx / this.Scale) + this.Offset);
```

Comment:
This (and below) changes the behaviour; I think the new behaviour is correct. I assume the perf consideration is irrelevant these days.

Reply:
Yes, this changes the behaviour if someone directly consumes the transformed values - otherwise you don't even notice this. But, as you also mentioned, I think it might have been more of an oversight that these methods just ignored the `Base`.
And yes, I would strongly assume that these functions get inlined by the compiler where it makes sense anyway.